helm/gateway-api-config/values.yaml

```diff
@@ -10,6 +10,11 @@ gateways:
   giantswarm-default:
     enabled: true
     name: giantswarm-default
+    className: giantswarm-default
+    service:
+      annotations:
+        external-dns.alpha.kubernetes.io/hostname: giantswarm-default.{{ $.Values.baseDomain }}
```

Comment:
I think this needs a comment.

Reply:
mh. maybe include the word `gateway` then?

Reply:
let's go with gateway for now as this is configurable. then we decide what to do for the wildcard and mimic the ingress behavior in https://github.com/giantswarm/giantswarm/issues/30161